beacon-chain/core/blocks/benchmarks/config.go

```diff
@@ -0,0 +1,48 @@
+package benchmarks
+
+import (
+	"fmt"
+
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+// ValidatorCount is for declaring how many validators the benchmarks will be
```

Comment:
Why do these need to be exported @0xKiwi?

Reply:
Any chance we could avoid this if we put this config file next to the place it's needed?

Reply:
It's needed in the `benchmarks` package and the `benchmark_files` package. They needed to be exported so the `benchmark_files` package can access what they are.

Reply:
Got it, that makes sense